src/Lean/Server/FileWorker/RequestHandling.lean

```diff
@@ -193,12 +202,26 @@ def locationLinksOfInfo (kind : GoToKind) (ictx : InfoWithCtx)
             | .const declName _ => do
               if ← ci.runMetaM i.lctx do Lean.Meta.isInstance declName then pure #[declName] else pure #[]
             | .app fn arg => do pure $ (← extractInstances fn).append (← extractInstances arg)
+            | .mdata _ e => extractInstances e
             | _ => pure #[]
           if let some instArg := appArgs.get? instIdx then
             for inst in (← extractInstances instArg) do
               results := results.append (← ci.runMetaM i.lctx <| locationLinksFromDecl i inst)
             results := results.append elaborators -- put elaborators at the end of the results
         return results
+    locationLinksDefault
+
+  match i with
+  | .ofDelabTermInfo { location? := some (module, range), .. } =>
+    let targetUri := (← documentUriFromModule rc.srcSearchPath module).getD doc.meta.uri
```

Comment:
Thanks. Now it falls through to the normal TermInfo handler, since I figure it's better to give the user something relevant to jump to than nothing at all. What do you think?

Reply:
Sounds good!